res/layout/list_item_menu.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is identical to the standard list item, but it has an ImageView on the right side. We can probably merge these.